docker/root/etc/cont-init.d/20-config

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/with-contenv bash
 
-# make folders
-mkdir -p /database
-
 # copy config
 if [ ! -f "/config/settings.json" ]; then
   cp -a /defaults/settings.json /config/settings.json
 fi
 
+# check if db exists
+if [ ! -f "/database/filebrowser.db" ]; then
+  touch "/database/filebrowser.db"
+fi
```

Comment:
This file will be automatically created by the filebrowser if it has write permissions for the mounted folder.